ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/KsqlResource.java

```diff
@@ -929,6 +948,22 @@ private void validateExplain(final PreparedStatement<Explain> statement) {
       explain(statement, scopedPropertyOverrides, ksqlConfig, executionSandbox);
     }
 
+    /**
+     * @deprecated `RUN SCRIPT` is deprecated since 5.2 and will be removed in the next major rel.
```

Comment:
nit: clarify that it's deprecated from the REST API, to avoid confusing someone who reads this and thinks it's deprecated from the CLI too.

Reply:
This code is in the rest api.  I don't think we need to call out such things here. The release and upgrade notes both explain this.